source/transactions/tests/error-labels.yml

```diff
@@ -959,3 +959,247 @@ tests:
       collection:
         data:
           - _id: 1
+
+  - description: do_not_add_unknown_commit_label_to_MaxTimeMSExpired_inside_transactions
+
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 1 }
+      data:
+        failCommands: ["aggregate"]
+        errorCode: 50  # MaxTimeMSExpired
+
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+          session: session0
+          document:
+            _id: 1
+        result:
+          insertedId: 1
+      - name: aggregate
+        object: collection
+        arguments:
+          pipeline:
+            - $project:
+                _id: 1
+          maxTimeMS: 60000
```

Comment:
Should this also be included in the corresponding CommandStartedEvent? As-is, I expect the matcher will just ignore it in the actual outgoing command as an extra field not present in the assertion. I imagine the value doesn't really matter, since the fail point likely triggers immediately, but it'd be good to test that the driver sends what we expect.

Reply:
Nice catch, done.